examples/tls.rs

```diff
@@ -42,6 +62,9 @@ async fn main() -> io::Result<()> {
     let domain = options.domain.unwrap_or(options.host);
     let topics = vec!["tester"];
 
+    let certs = load_certs(&options.cert)?;
```

Comment:
@hgm-king  do we want to allow support for default paths ( typically os specfiic)?
example :  https://github.com/kafka-rust/kafka-rust/blob/master/examples/example-ssl.rs#L46-L55